packages/sessions-sdk-react/src/components/token-list.tsx

```diff
@@ -163,9 +171,10 @@ const LoadingToken = () => (
       </div>
     </div>
     <div className={styles.amountAndActions}>
-      <div className={styles.amountAndSymbol}>
+      <div className={styles.amountAndDetails}>
         <span className={styles.amount} />
         <span className={styles.symbol} />
```

Comment:
You need to remove this line so the loader matches the loaded state properly and you no longer have this span in the loaded state